src/test/java/net/smartcosmos/usermanagement/role/persistence/RolePersistenceServiceTest.java

```diff
@@ -129,6 +129,58 @@ public void thatUpdateRoleSucceeds() {
     }
 
     @Test
+    public void thatUpdateRoleSucceedsWithSameName() {
```

Comment:
I think you should call this `thatUpdateRoleAddingAuthoritySucceeds` and the current `thatUpdateRoleSucceeds` should be `thatUpdateRoleNameSucceeds`.